platform/kobo/release-ip.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh
 
+export PATH=$PATH:/sbin
```

Comment:
So yeah, I actually wrote that 2 weeks ago. I didn't realize you have to take two steps to publish the comment. Just a simple matter of

1) reminding yourself in a year
2) making sure people don't accidentally remove it if it does something useful.